lua/indentmini/init.lua

```diff
@@ -58,10 +45,27 @@ local function indentline()
     ctx[row] = indent
 
     local shiftw = indent_step(bufnr)
+    local last_defined_level = 0
     for i = 1, indent - 1, shiftw do
-      local hi_name = 'IndentLine'
-      local iteration = math.floor((i - 1) / shiftw) + 1
-      hi_name = string.format('%s%d', hi_name, (iteration - 1) % #hl_groups + 1)
+      local indent_level = math.floor((i - 1) / shiftw) + 1
+      local hi_name = ('IndentLine%d'):format(indent_level)
+      -- Attempt to fetch highlight details for the current level
+      local hl_details = api.nvim_get_hl(0, { name = hi_name })
```

Comment:
there has default = true in nvim_set_hl . if there already have configed it will use that. so don't need get hl first :)

Reply:
no need bound theme into setup then colorscheme can handle it .

Reply:
I'm sorry, I'm not understanding what I should do.